pkg/api/validation/validation.go

```diff
@@ -677,6 +677,51 @@ func validateISCSIVolumeSource(iscsi *api.ISCSIVolumeSource, fldPath *field.Path
 	return allErrs
 }
 
+func validateISCSIPersistentVolumeSource(iscsi *api.ISCSIPersistentVolumeSource, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if len(iscsi.TargetPortal) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("targetPortal"), ""))
+	}
```

Comment:
Dont we need a field check on `portals` ?

Reply:
I assume this is a duplicate of validateISCSIVolumeSource, and as such, we cannot tighten existing validation

Reply:
@liggitt thanks, comments came out of order, missed your previous comment.